source/guide_pleroma.rst

```diff
@@ -0,0 +1,279 @@
+ CREATE ROLE
+ CREATE DATABASE
+ You are now connected to database "pleroma" as user "isabell".
+ CREATE EXTENSION
+ CREATE EXTENSION
+ CREATE EXTENSION
+ [isabell@stardust ~]$
+
+Database migration
+------------------
+
+.. note:: For these next steps, the default will be to run pleroma using the dev configuration file, ``config/dev.secret.exs``. To run them using the prod config file, prefix each command at the shell with ``MIX_ENV=prod``. For example: ``MIX_ENV=prod mix phx.server``. Documentation for the config can be found at docs/config.md in the repository, or at the "Configuration" page on https://docs-develop.pleroma.social/config.html
+
+Make a copy of the file ``generated_config.exs`` in the directory ``~/pleroma/config`` and rename it to ``dev.secret.exs``. ``cd`` the root directory of Pleroma and run ``mix ecto.migrate`` to migrate the database.
+
+::
+
+ [isabell@stardust ~]$ cp ~/pleroma/config/generated_config.exs ~/pleroma/config/dev.secret.exs
+ [isabell@stardust ~]$ cd ~/pleroma
+ [isabell@stardust ~]$ mix ecto.migrate
+ [isabell@stardust ~]$
+
+Configure web server
+--------------------
+
+.. note:: Pleroma is running on port 4000 in the default configuration.
+
+To access the front end you have to set a new backend. You need the domain and the port you configured earlier.
+
+.. include:: includes/web-backend.rst
+
+As example for this guide:
+
+::
+
+ [isabell@stardust ~]$ uberspace web backend set isabell.uber.space --http --port 4000
+ Set backend for isabell.uber.space/ to port 4000; please make sure something is listening!
+ You can always check the status of your backend using "uberspace web backend list".
+ [isabell@stardust ~]$
+
+.. Comment out because its not necessary
+.. Start Pleroma
+ =============
+
+ With ``mix phx.server`` you can start Pleroma in the debug mode.
+
+ ::
+
+  [isabell@stardust ~] cd ~/pleroma
+  [isabell@stardust ~] mix phx.server
+
+ To stop the service type an ``a`` in the terminal and press Enter.
+
+Setup deamon
+============
+
+Create ``~/etc/services.d/pleroma.ini`` with the following content and make sure to replace ``<username>`` with your Uberspace username:
+
+.. code-block:: ini
+
+ ; Assumes pleroma is installed in /home/<username>/pleroma and running as the pleroma user
+ ; Also assumes mix is in /usr/bin, this might differ on BSDs or niche Linux distros
+ ; Logs into /home/<username>/logs
+ [program:pleroma]
+ command=/usr/bin/mix phx.server
+ directory=/home/<username>/pleroma
+ autostart=true
+ autorestart=true
+ user=<username>
+ environment =
+   MIX_ENV=dev,
+   HOME=/home/<username>,
+   USER=<username>,
+   PATH="/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/home/<username>/bin:%(ENV_PATH)s",
+   PWD=/home/<username>/pleroma
+ stdout_logfile=/home/<username>/logs/stdout.log
+ stdout_logfile_maxbytes=50MB
+ stdout_logfile_backups=10
+ stderr_logfile=/home/<username>/logs/stderr.log
+ stderr_logfile_maxbytes=50MB
+ stderr_logfile_backups=10
```

Comment:
Its required to save the logs at ~/logs because this directory is not set in the default.

Reply:
I guess that it would confuse users to do this different than all the other lab guides. For acessing them you `supervisorctl tail` should be used anyway